README Detailed.md

```diff
@@ -0,0 +1,302 @@
+# OpenCortex
```

Comment:
You can leave this one out. Just the README is enough.

Reply:
Fixed in commit